### ISSUE TITLE
Re-open "ExperienceReplayBuffer stores second-last transition twice"

### ISSUE THREAD

**Issue by Phlogiston90** [Shmuma/ptan#26]
Re-open of #25 .

**Comment by Shmuma** [comment]
Are you sure you're using the correct version? The fix is in the `torch-1.0.1` branch, and not yet merged to master. 

The output of the code from #25:
```
(ptan) ShmumaMac:ptan shmuma$ git status
On branch torch-1.0.1
Your branch is up to date with 'origin/torch-1.0.1'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	t1.py

nothing added to commit but untracked files present (use "git add" to track)
(ptan) ShmumaMac:ptan shmuma$ python t1.py 
episode over: step 5: (total_reward, steps) = (10.0, 4)
episode over: step 9: (total_reward, steps) = (10.0, 4)
episode over: step 13: (total_reward, steps) = (10.0, 4)
episode over: step 17: (total_reward, steps) = (10.0, 4)

ExperienceFirstLast(state=2, action=1, reward=3.0, last_state=3)
ExperienceFirstLast(state=3, action=0, reward=4.0, last_state=None)
ExperienceFirstLast(state=0, action=1, reward=1.0, last_state=1)
ExperienceFirstLast(state=1, action=1, reward=2.0, last_state=2)
ExperienceFirstLast(state=2, action=1, reward=3.0, last_state=3)
ExperienceFirstLast(state=3, action=1, reward=4.0, last_state=None)
ExperienceFirstLast(state=0, action=1, reward=1.0, last_state=1)
ExperienceFirstLast(state=1, action=0, reward=2.0, last_state=2)
ExperienceFirstLast(state=2, action=1, reward=3.0, last_state=3)
ExperienceFirstLast(state=3, action=1, reward=4.0, last_state=None)
```
You can also check new test which has the same logic as your code: https://github.com/Shmuma/ptan/blob/torch-1.0.1/tests/test_experience.py#L154

**Comment by Phlogiston90** [comment]
Sorry you are right! I checked out the master branch by mistake! Thanks for fixing!
